eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/coding/checkstyle-metadata.properties

```diff
@@ -188,3 +188,10 @@ MoveVariableInsideIfCheck.desc = Checks if a variable is only used inside if sta
 
 RequireFailForTryCatchInJunitCheck.name = Require Fail For Try/Catch in JUnit Check
 RequireFailForTryCatchInJunitCheck.desc = Checks if a try/catch block has a junit fail assertion inside the try  for a junit method.
+
+Jsr305AnnotationsCheck.name=Jsr305AnnotationsCheck
+Jsr305AnnotationsCheck.desc=Checks method parameters and return values for the presence of @Nonnull, @Nullable, or @CheckForNull annotations.
+Jsr305AnnotationsCheck.packages=Packages to be checked.
+Jsr305AnnotationsCheck.excludePackages=Packages excluded from checking.
```

Comment:
Why do we need a package property and an exclude package property?
Why isn't package enough?
I also think the descriptions should be more descriptive. Are we excluding all classes in the package from all validation? Will it include sub-packages?

Reply:
You're right of course. I have added some more information here.